cmd/collectors/restperf/restperf.go

```diff
@@ -1545,6 +1544,9 @@ func (r *RestPerf) handleError(err error, href string) (map[string]*matrix.Matri
 		// the table or API does not exist. return ErrAPIRequestRejected so the task goes to stand-by
 		return nil, fmt.Errorf("polling href=[%s] err: %w", href, errs.New(errs.ErrAPIRequestRejected, err.Error()))
 	}
+	// if errs.IsRestErr(err, errs.CMReject) {
+	// 	return nil, fmt.Errorf("CM reject href=[%s] err: %w", href, errs.New(errs.ErrNoInstance, err.Error()))
+	// }
```

Comment:
Is this required ?

Reply:
good catch. This was from an earlier dev on this problem. Removed since the collector handles errs.CMReject